### PR TITLE
[KBFS-3331] Widget dropdown menu should say "Open folders in Finder"

### PR DESCRIPTION
**Pull request by brayoh**
#### Description of task:
Widget drop-down menu should say "Open folders in Finder" and it should open in Finder instead of opening the app.

#### Relevant Screenshots
screenshot 2018-09-27 at 20 26 43

### CODE REVIEW

File: shared/menubar/index.desktop.js [hunk 1]
```diff
@@ -151,7 +152,7 @@ class MenubarRender extends React.Component<Props, State> {
       },
       'Divider',
       ...(this.props.loggedIn ? [{title: 'Open main app', onClick: () => this.props.openApp()}] : []),
-      {title: 'Open files', onClick: () => this.props.openApp(Tabs.fsTab)},
+      {title: 'Open folders in finder', onClick: () => this.props.showInFinder('/')},
```
Comment (jzila) [inline shared/menubar/index.desktop.js]: Oh wait, this should actually say the appropriate FileUI name per operating system.

Comment (brayoh) [reply shared/menubar/index.desktop.js]: Cool, let me find out how to resolve that.

**Comment by brayoh** [comment]
@jzila I've made the change

File: shared/menubar/index.desktop.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import * as Tabs from '../constants/tabs'
 import * as Styles from '../styles'
 import ChatContainer from './chat-container.desktop'
 import FilesPreview from './files-container.desktop'
-import {isDarwin} from '../constants/platform'
+import {isDarwin, isWindows} from '../constants/platform'
```
Comment (jzila) [inline shared/menubar/index.desktop.js]: This will now fail lint, since you aren't using `isWindows` any more.

**Pull request merged**